src/containers/JudgingPanel.js

```diff
@@ -139,7 +142,13 @@ export default () => {
   }
 
   const sync = () => {
-    return syncToFirebase(projects, setMessage)
+    if (
+      window.confirm(
+        `Are you sure you want to sync ${projects.length} projects? This is a "very" destructive action`
+      )
```

Comment:
LMAO thank you

Reply:
dont want other peeps to press it accidentally LOL i pressed it accidentally when testing and regretted it